src/psyclone/domain/lfric/lfric_builtins.py

```diff
@@ -42,14 +42,14 @@
     The LFRicBuiltInCallFactory creates the Python object required for
     a given built-in call. '''
 
-from __future__ import absolute_import
 import abc
 from psyclone.core.access_type import AccessType
 from psyclone.errors import InternalError
 from psyclone.psyGen import BuiltIn
-from psyclone.psyir.symbols import DataSymbol, INTEGER_SINGLE_TYPE
-from psyclone.psyir.nodes import Assignment, Reference, StructureReference, \
-    BinaryOperation
+from psyclone.psyir.symbols import (DataSymbol, INTEGER_SINGLE_TYPE,
```

Comment:
Module imports in alphabetical order, please 😃

Reply:
Done.